cmd/argo/commands/install.go

```diff
@@ -339,6 +341,36 @@ func installUi(clientset *kubernetes.Clientset, args InstallFlags) {
 	}
 }
 
+func installUIService(clientset *kubernetes.Clientset, args InstallFlags) {
+	svcName := "argo-ui"
```

Comment:
Can you make this a constant?

Reply:
Oops, I created the constant ArgoServiceName but I forgot to use it. Thanks for catching this.